### ISSUE TITLE
Password box refreshing?

### ISSUE THREAD

**Issue by nzkfc** [Its-Scru/scru_serverlock#4]
We mostly use this on our dev environment as we have sporadic players joining to test stuff or similar.

When the password box comes up, we have to spam type 2 times to then be able to enter a password, otherwise the box refreshes and clears while typing.

I put it on our production city while doing some update testing before launching and noticed it was almost impossible due to I assume our connection queue or Pheonix Anti-cheat box that appears when connecting causing this one to constantly clear, so you have to copy the password to clipboard then spam Ctrl+V while hitting submit with the mouse haha.

Any way that can be fixed?

**Comment by Its-Scru** [comment]
I will check it out.

**Comment by Its-Scru** [comment]
Yeah it must be your connection queue or the anti-cheat that is interfering with the password box. At the moment, I'm not sure on a fix.
